tensorflow_datasets/structured/breast_cancer.py

```diff
@@ -0,0 +1,123 @@
+}
+"""
+
+
+_DESCRIPTION = """\
+Features are computed from a digitized image of a fine needle aspirate (FNA)
+of a breast mass. They describe characteristics of the cell nuclei present in
+the image. The goal of the dataset is to predict whether the cancer in the cell
+is benign or malignant. The attribute information is as follows:
+#  Attribute                     Domain
+   -- -----------------------------------------
+   1. Sample code number            id number
+   2. Clump Thickness               1 - 10
+   3. Uniformity of Cell Size       1 - 10
+   4. Uniformity of Cell Shape      1 - 10
+   5. Marginal Adhesion             1 - 10
+   6. Single Epithelial Cell Size   1 - 10
+   7. Bare Nuclei                   1 - 10
+   8. Bland Chromatin               1 - 10
+   9. Normal Nucleoli               1 - 10
+  10. Mitoses                       1 - 10
+  11. Class:                        Benign or Malignant
+  For this dataset, 0 represents benign and 1 represents malignant
+  """
+
+
+def convert_to_int(number):
+  return -1 if number == "?" else np.int8(number)
+
+
+COLUMNS = ["clump_thickness", "uniformity_of_cell_size",
+           "uniformity_of_cell_shape", "marginal_adhesion",
+           "single_epithelial_cell_size", "bare_nuclei", "bland_chromatin",
+           "normal_nucleoli", "mitoses"]
+
+
+FEATURES = tfds.features.FeaturesDict({
+    "clump_thickness": tf.int8,
+    "uniformity_of_cell_size": tf.int8,
+    "uniformity_of_cell_shape": tf.int8,
+    "marginal_adhesion": tf.int8,
+    "single_epithelial_cell_size": tf.int8,
+    "bare_nuclei": tf.int8,
+    "bland_chromatin": tf.int8,
+    "normal_nucleoli": tf.int8,
+    "mitoses": tf.int8
+})
+
+
+class BreastCancer(tfds.core.GeneratorBasedBuilder):
+  """Breast Cancer Wisconsin dataset."""
+
+  VERSION = tfds.core.Version("0.0.1")
+  def _info(self):
+    return tfds.core.DatasetInfo(
+        builder=self,
+        description=_DESCRIPTION,
+        features=tfds.features.FeaturesDict({
+            "features": FEATURES,
+            "label": tfds.features.ClassLabel(names=['Benign', 'Malignant'])
+        }),
+        supervised_keys=("features", "label"),
+        homepage="https://archive.ics.uci.edu/ml/datasets/Breast+Cancer+Wisconsin+(Diagnostic)",
+        citation=_CITATION,
+    )
+
+  def _split_generators(self, dl_manager):
+    """Generate Splits"""
+    bcwd_file = dl_manager.download(BCWD_URL)
+    all_lines = tf.io.gfile.GFile(bcwd_file).read().split("\n")
+    records = [l.split(",") for l in all_lines if l]
+
+    # Specify the splits
+    return [
+        tfds.core.SplitGenerator(
+            name=tfds.Split.TRAIN,
+            gen_kwargs={"records": records}),
+    ]
+
+  def _generate_examples(self, records):
+    for i, row in enumerate(records):
+      label = row[-1]
+      #the 0 label represents 2 in the original dataset(benign) & 1 represents
+	 # 4(malignant)
```

Comment:
nit: comment is not aligned.

Reply:
@cyfra Thanks for approving the pull request. The comment is now aligned